llvm/include/llvm/ExecutionEngine/Orc/Core.h

```diff
@@ -1453,7 +1453,8 @@ class ExecutionSession {
 
   /// Construct an ExecutionSession with the given ExecutorProcessControl
   /// object.
-  ExecutionSession(std::unique_ptr<ExecutorProcessControl> EPC);
+  ExecutionSession(std::unique_ptr<ExecutorProcessControl> EPC,
+                   std::shared_ptr<SymbolStringPool> SSP = nullptr);
```

Comment:
If we keep the default value here, we don't need the second patch. It would reduce the PR size significantly: https://github.com/llvm/llvm-project/pull/85056/commits/23a9ec6ce3fa87e0cdaabab420d9f9938f246d0d

I am not sure though, do we want default values in the public headers?